app/views/tagging_spreadsheets/index.html.erb

```diff
@@ -20,9 +20,20 @@
       <tr>
         <td><%= state_label_for(label_type: spreadsheet.label_type,
                                 title: spreadsheet.state_title,
-                                data_attributes: spreadsheet.data_attributes) %></td>
-        <td><%= spreadsheet.url %></td>
-        <td><%= spreadsheet.description %></td>
+                                data_attributes: []) %></td>
```

Comment:
Same as above, we can remove this parameter here and delete code from the helper.